addons/overpressure/functions/fnc_firedEHOP.sqf

```diff
@@ -0,0 +1,21 @@
+/*
+ * Author: joko // Jonas
+ *
+ * Handle fire of Other Weapons
+ *
+ * Argument:
+ * 0: Unit that fired (Object)
+ * 1: Weapon fired (String)
+ * 2: Muzzle (String)
+ * 3: Mode (String)
+ * 4: Ammo (String)
+ * 5: Magazine (String)
+ * 6: Projectile (Object)
+ *
+ * Return value:
+ * None
+ */
+
+ if (missionNameSpace getVariable [(QGVAR(Damage) + _this select 1),(([_this select 1,_this select 5] call DFUNC(cacheOverPressureVales)) select 2)]) then {
```

Comment:
This should be divided. Because of how SQF works, the second term of the array is evaluated regardless if the value is already cached or not, so performance is wasted nonetheless.

Typically you need to do:

```sqf
_varName = (QGVAR(Damage) + _this select 1);
if (isNil _varName) then {
   ...
} else {
   ...
};